### ISSUE TITLE
labeling regression: TeX labels don't render properly in two vectors perp example

### ISSUE THREAD

**Issue by hypotext** [penrose/penrose#516]
**Describe the bug**
TeX labels render as strings without typesetting:
![image](https://user-images.githubusercontent.com/2762356/110256902-550ad900-7f69-11eb-96a7-f67d59ffc583.png)

In the Shape JSON, the TeX string appears to not be going through (not sure why underscores disappeared):

```
string: Object
tag: "StrV"
contents: "x2"
```

I checked the Style and Substance and would expect the subscript `$x_1$` to render.

**To Reproduce**
On latest main (f03b84c18e60a76b52f7cb9ab003b9022f5b7e33), run the wiki path `roger watch linear-algebra-domain/twoVectorsPerp-unsugared.sub linear-algebra-domain/linear-algebra-paper-simple.sty linear-algebra-domain/linear-algebra.dsl`.

**Expected behavior**
I expect the image to look as it is on the wiki.

![image](https://user-images.githubusercontent.com/2762356/110256961-ab781780-7f69-11eb-9a43-d8d6970af257.png)

**Next steps**
It would be good for someone to do a git bisect to see which commit caused this. Alternatively, if our automated tests can do a visual diff, that would also be helpful to catch these things.

**Comment by wodeni** [comment]
Label statements are evaluated sequentially. The correct program should be:
```
VectorSpace X
Vector x1 
In(x1, X)
Vector x2 
In(x2, X)
Unit(x1)
Orthogonal(x1, x2)
AutoLabel All
Label x1 $x_1$
Label x2 $x_2$
```

**Comment by hypotext** [comment]
Thanks for catching it! This appears to have been introduced in an earlier commit: https://github.com/penrose/penrose/blame/main/examples/linear-algebra-domain/twoVectorsPerp-unsugared.sub

Reopening as the issue hasn't been fixed :) I can fold it into my existing bugfix PR.

**Comment by wodeni** [comment]
> Reopening as the issue hasn't been fixed :) I can fold it into my existing bugfix PR.

Sure thing!

**Comment by hypotext** [comment]
Fixed in b1013e913ba1f59b00123e98b8e68d2848761866
